MIGRATIONS.md

```diff
@@ -3,6 +3,17 @@
 This file documents changes in the data model. Please explain any changes to the
 data model as well as any custom migrations.
 
+## WordPress 46 (@aerych 2016-01-29)
+
+- `BlogSettings` added string `sharingButtonStyle`. Stores style to use for sharing buttons.
+- `BlogSettings` added string `sharingLabel`. Stores the text to show in the sharing label. 
+- `BlogSettings` added string `sharingTwitterName`. Stores the username used when sharing to Twitter.
+- `BlogSettings` added bool `sharingCommentLikesEnabled`. Whether comments display a like button.
+- `BlogSettings` added bool `sharingDisabledLikes`.  Whether posts display a like button. 
+- `BlogSettings` added bool `sharingDisabledReblogs`. Whether posts display a reblog button. 
```

Comment:
Now i don't feel *THAT* bad about the `BlogSettings` entity. Thanks for the moral support Eric!